domains/dns.haruki.is-cool.dev.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "Adguard secure DNS",
+
+    "domain": "is-cool.dev",
+    "subdomain": "dns.haruki",
+
+    "owner": {
+        "repo": "https://github.com/haruki-py/register-open",
+        "email": "aanandkrishna344@gmail.com"
+    },
+
+    "record": {
+        "A": ["103.161.55.146"]
```

Comment:
Can't reach your site.

Reply:
I have to proxy it to make it accessible. Can only be done after merge